tests/example-all-fail/CMakeLists.txt

```diff
@@ -0,0 +1,71 @@
+# Get the exercise name from the current directory
+get_filename_component(exercise ${CMAKE_CURRENT_SOURCE_DIR} NAME)
```

Comment:
The `CMakeLists.txt` files had replaced `${CMAKE_CURRENT_SOURCE_DIR}` with `bob`. I've reverted this to better simulate what happens with actual exercises.